tools/tests/rustls_header_version.rs

```diff
@@ -15,27 +11,10 @@ use tree_sitter::{Parser, Query, QueryCursor};
 /// currently has an msrv of 1.73.
 #[test]
 fn rustls_header_versions_match() {
-    // Parse Cargo.toml as a generic TOML Table.
-    let mut metadata_file =
-        File::open(PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../librustls/Cargo.toml"))
-            .expect("failed to open Cargo.toml");
-    let mut metadata_content = String::new();
-    metadata_file
-        .read_to_string(&mut metadata_content)
-        .expect("failed to read Cargo.toml");
-    let metadata = metadata_content.parse::<Table>().unwrap();
-
-    // Find the crate version specified in Cargo.toml
-    let package_metadata = metadata["package"]
-        .as_table()
-        .expect("missing package metadata");
-    let crate_version = package_metadata["version"]
-        .as_str()
-        .expect("missing crate version");
-
     let version_in_header = version_in_header();
     assert_eq!(
-        crate_version, version_in_header,
+        env!("CARGO_PKG_VERSION"),
```

Comment:
unfortunately this is a different package to librustls.

Reply:
Reverted that change for now.